crates/nu-command/src/filters/default.rs

```diff
@@ -133,12 +192,20 @@ fn default(
 
 #[cfg(test)]
 mod test {
-    use super::*;
 
     #[test]
     fn test_examples() {
+        use super::*;
         use crate::test_examples;
 
         test_examples(Default {})
     }
+
+    #[test]
```

Comment:
_integration_ tests like this are typically not done in the module of the command directly in Nushell :open_mouth: 

i think it would be best to move this test to `crates/nu-command/tests/commands/default.rs` :wink: